drivers/scheduler/k8s/k8s.go

```diff
@@ -1805,10 +1805,26 @@ func (k *K8s) GetVolumes(ctx *scheduler.Context) ([]*volume.Volume, error) {
 }
 
 //ResizeVolume  Resize the volume
-func (k *K8s) ResizeVolume(ctx *scheduler.Context) ([]*volume.Volume, error) {
+func (k *K8s) ResizeVolume(ctx *scheduler.Context, configMapName string) ([]*volume.Volume, error) {
 	k8sOps := k8s_ops.Instance()
 	var vols []*volume.Volume
 	for _, spec := range ctx.App.SpecList {
+		// Add security annotations if running with auth-enabled
+		if configMapName != "" {
+			configMap, err := k8sOps.GetConfigMap(configMapName, "default")
```

Comment:
are you sure namespace will be always `default`?

Reply:
Yes we create the config maps in the jenkins job and don't want the users to need to specify the namespace.